### ISSUE TITLE
When Searching on the inserter down arrow keys should work

### ISSUE THREAD

**Issue by mtias** [WordPress/gutenberg#3849]
When searching on the inserter, it should be possible to access the block results using the arrow keys. Right now you can only access them with a tab press.

**Comment by afercia** [comment]
During WCUS contributor day the inserter was tested in depth by Jackie D’Elia (@jdelia) and Katherine White (@katwhite). One of their findings was to consider to use the down arrow key also for moving the focus from the tabs to the panel below, since on the tabs users are already using arrows.

It would make sense to pair that with what's proposed here for the search field.

**Comment by mtias** [comment]
Oh, cool, I thought that was decided against before. I definitely agree with allowing it to go to the tabs as well.

**Comment by afercia** [comment]
I spoke too soon 🙂  Actually the feedback received during WCUS contributor day a11y testing was the opposite. Big thanks to Jackie D’Elia (@jdelia) and Katherine White (@katwhite) who tested all the day, see the [feedback summary](https://gist.github.com/kwhite/22cf005d0a23f3c66dd8df62a6bde574).

The relevant points for this specific issue are:
1. Leaving the search bar should be a tab action, as it is now.
2. Navigating INTO a Section should be the down arrow.

I wouldn't be opposed to 2 in addition to Tab. Reasoning:
- when searching, you're just typing and Tab is the default, expected way to move to the next focusable element
- instead, when switching through tabs, you're already using the arrow keys: it would be handy to be able to use also the Down arrow to jump to the active tab panel (note: it's not standard as per the ARIA Authoring Practices, but I recognize it's something worth trying)

**Comment by mcsf** [comment]
Just to be clear, can you confirm all these?

- When focusing a tab (Recent, Blocks, &c.), `↓` should move us to the panel below, but `↑` shouldn't move us to search. `→` and `←` should still navigate through tabs, as today.
- When in a panel, the current behavior should be retained where `→` and `↓` work the same way (and, respectively, `←` and `↑`).

**Comment by afercia** [comment]
@mcsf that would be perfect, thanks!
